pages/blog/event-driven-api-documentation-made-simple-clientside-rendering.md

```diff
@@ -0,0 +1,219 @@
+---
+title: Event driven API documentation made simple (Client-Side Rendering).
+date: 2022-08-22T9:43:15+01:00
+featured: true
+type: Engineering
+tags: ['documentation','guide']
+cover: /img/posts/event-driven-API-documentation-made-simple/documentation.webp
+authors:
+  - name: Michal Gornicki
+    photo: /img/avatars/michalgornicki.webp
+    link: https://www.linkedin.com/in/michal-gornicki-83865161/
+excerpt: A simple guide for programmers looking for instructions on how to generate documentation using their AsyncAPI files.
+---
+
+> This post originally appeared on [Michal's Corner](https://michals-corner.vercel.app/blog/event-driven-documentation)
+
+This guide is directed toward folks looking for instructions on how to generate documentation using their AsyncAPI files. Event-driven APIs are not the same as the synchronous APIs you usually document with OpenAPI or GraphQL. Many people use AsyncAPI now, and it is time to provide the community with a guide that shows what options the community has to render documentation on the client-side.
+
+Oh, if you are still not sure about Client-Side Rendering or Server-Side Rendering, then you can read all about it here 👉 [Website rendering for beginners](https://michals-corner.vercel.app/blog/website-rendering-for-beginners) (shameless plug 😉).
+
+I will cover the usage for:
+
+- [React](#react)
+- [Vue](#vue)
+- [Web Components](#wc)
+- [Standalone Bundle](#sb)
+
+All examples will use this same AsyncAPI sample file 👇
+
+```
+{
+  "asyncapi": "2.2.0",
```

Comment:
Better option will be point to the latest version of asyncapi:

```suggestion
  "asyncapi": "2.4.0",
```

It would be nice to add information that the spec is used as JSON to make it easier to use on the JS side :) Small suggestion, you don't need to add it.

Reply:
@magicmatatjahu that's done